src/main/java/jenkins/branch/NoTriggerOrganizationFolderProperty.java

```diff
@@ -72,28 +68,17 @@ public String getDisplayName() {
             return Messages.NoTriggerBranchProperty_suppress_automatic_scm_triggering();
         }
 
-        @Override
-        public boolean isApplicable(Class<? extends AbstractFolder> containerType) {
-            return super.isApplicable(containerType) && legacyCodeActive();
-        }
-
         public FormValidation doCheckBranches(@QueryParameter String value) {
             try {
                 Pattern.compile(value);
-                return FormValidation.warning(Messages.NoTriggerOrganizationFolderProperty_PropertyMigration());
+                return FormValidation.ok();
             } catch (PatternSyntaxException x) {
                 return FormValidation.error(x.getMessage());
             }
         }
 
     }
 
-    static boolean legacyCodeActive() {
```

Comment:
This was being used to hide descriptors once basic-branch-build-strategies was installed, even though the migration didn't happen after the other changes. Thanks to @kshultzCB for catching this!